### PR TITLE
feat(google-analytics-4 client): add the base code for the ga4 for openebs engines

### PR DESCRIPTION
**Pull request by Abhinandan-Purkait**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module github.com/openebs/go-ogle-analytics
```
Comment (niladrih) [inline go.mod]: This module name needs to change..

**Review by niladrih — COMMENTED** [review]
Left one comment

File: go.mod (hunk 1 above)
Comment (Abhinandan-Purkait) [reply go.mod]: Done.

**Review by avishnu — APPROVED** [review]
lgtm

File: README.md [hunk 2]
```diff
@@ -1,2 +1,82 @@
...
+## Google Analytics 4
+
+Track and monitor your Go programs for free with Google Analytics
+
+The `ga` package is essentially a Go wrapper around the [Google Analytics - Measurement Protocol (Google Analytics 4)](https://developers.google.com/analytics/devguides/collection/protocol/ga4/reference?client_type=gtag)
+
+### Install
+
+```
+go get -v github.com/openebs/go-ogle-analytics
```
Comment (tiagolobocastro) [inline README.md]:
```suggestion
go get -v github.com/openebs/google-analytics-4
```
?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,82 @@
...
+2. Create a `ga-test.go` file
+
+	``` go
+	package main
+
+        import (
+            "fmt"
+
+            gaClient "github.com/openebs/go-ogle-analytics/client"
+            gaEvent "github.com/openebs/go-ogle-analytics/event"
```
Comment (tiagolobocastro) [inline README.md]:
```suggestion
            gaClient "github.com/openebs/google-analytics-4/client"
            gaEvent "github.com/openebs/google-analytics-4/event"
```

**Review by niladrih — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (Abhinandan-Purkait) [reply README.md]: Missed this, thanks.

**Review by tiagolobocastro — APPROVED** [review]

**Pull request merged**